apps/backend/src/mcp/tools/device_info.py

```
@@ -203,6 +200,775 @@ def _parse_smart_output(smart_output: str, drive_path: str) -> Dict[str, Any]:
+            }
+
+    except Exception as e:
+        connectivity_results["ssh"] = {"status": "error", "error": str(e)}
+        logger.warning(f"SSH test failed for {device}: {e}")
+
+    return connectivity_results
+
+
+async def _analyze_system_metrics(
+    ssh_client, connection_info: SSHConnectionInfo
+) -> Dict[str, Any]:
```

Ruff (0.12.2)

Use `dict` instead of `Dict` for type annotation

Replace with `dict`

(UP006)